vassal-app/src/main/java/VASSAL/build/module/map/DrawPile.java

```diff
@@ -751,6 +824,51 @@ else if (FACE_UP_MESSAGE.equals(key)) {
     else if (FACE_DOWN_MESSAGE.equals(key)) {
       dummy.setFaceDownMessage((String)value);
     }
+    else if (FACE_UP_HOTKEY.equals(key)) {
+      if (value instanceof String) {
+        value = NamedHotKeyConfigurer.decode((String) value);
+      }
+      dummy.setFaceUpKey((NamedKeyStroke) value);
```

Comment:
What's with all the assigning to `dummy`?

Reply:
That's just the way the Deck / Draw Pile unholy sandwich works. DrawPile is a game component and holds the fields and saves/loads them. Deck is the runtime element that's "sort of a GamePiece" and "sort of a Stack" and stuff.